src/detect-fast-pattern.c

```diff
@@ -319,8 +319,7 @@ static int DetectFastPatternSetup(DetectEngineCtx *de_ctx, Signature *s, const c
             goto error;
         }
         uint16_t offset;
-        if (StringParseUint16(&offset, 10, 0,
-                              (const char *)arg_substr) <= 0) {
+        if (StringParseUint16(&offset, 10, 0, (const char *)arg_substr) <= 0) {
```

Comment:
unrelated format change wrt commit ?

Reply:
`clang-format` as this line was changed.

Reply:
Yeah, I see it now.  `clang-format rewrite-branch` got it wrong.